src/styles/utilis/_rest.scss

```diff
@@ -0,0 +1,18 @@
+html,
+body,
+div {
```

Comment:
Use classes where applicable, you already have .calendar available

Reply:
This is CSS reset and it's done this way as per my understanding